_posts/2016-01-15-GitHubAnimator.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: post
+name: GitHubAnimator
+title: 'GitHubAnimator'
+date: 2016-01-15 00:20:00 +00:00
+author: Pedro Santos
+image:
+    src: /assets/img/custom/blog/octokit.png
+tags:
+- GitHub 
+- animation
+- presentations
+- automation
+---
+
+### The Beginning
+Inspired by my friend [@SamirTalwar](https://twitter.com/SamirTalwar) and the initiative he set upon himself to [automate the world](https://noodlesandwich.com/]) I picked up on an idea I had many months ago. Automating the generation of presentations using GitHub commits.
```

Comment:
I've removed the broken link and replaced with a link to the blog post about 'automating everything'